vassal-app/src/main/java/VASSAL/tools/version/Version.java

```diff
@@ -1,96 +0,0 @@
-/*
```

Comment:
It's not among the classes in the the dependency report I ran last... This is _probably_ safe. Probably.